### PR TITLE
[Docs] update Japanese translation of isp_flashing_guide.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/isp_flashing_guide.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

**Review by s-show — APPROVED** [review]

File: docs/ja/isp_flashing_guide.md [hunk 1]
```diff
@@ -77,8 +78,8 @@ Teensy と Pro Micro のコントローラを ISP プログラマとして使用
 
 QMK ツールボックスは、このほとんど（すべて）に使用することができます。
 
-ただし、Teensy 2.0 ボードを使っている場合は、[Teensy Loader](https:/www.pjrc.comteensyloader.html) を使えば、Teensy 2.0 ボードに書き込むことができます。
-あるいは、`avrdude` (`qmk_install.sh` の一部としてインストールされています) や、[AVRDUDESS](https:/blog.zakkemble.netavrdudess-a-gui-for-avrdude)(Windows 用) を使って、Pro Micro に書き込んだり、ISP を書き込んだりすることができます。
+ただし、Teensy 2.0 ボードを使っている場合は、[Teensy Loader](https://www.pjrc.comteensyloader.html) を使えば、Teensy 2.0 ボードに書き込むことができます。
+あるいは、`avrdude` (`qmk_install.sh` の一部としてインストールされています) や、[AVRDUDESS](https://blog.zakkemble.netavrdudess-a-gui-for-avrdude)(Windows 用) を使って、Pro Micro に書き込んだり、ISP を書き込んだりすることができます。
```
Comment (shelaf) [inline docs/ja/isp_flashing_guide.md:82]: 😆
```suggestion
あるいは、`avrdude` (`qmk_install.sh` の一部としてインストールされています) や、[AVRDUDESS](https://blog.zakkemble.netavrdudess-a-gui-for-avrdude/)(Windows 用) を使って、Pro Micro に書き込んだり、ISP を書き込んだりすることができます。
```

**Review by shelaf — APPROVED** [review]

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**